tests/backup_test.php

```diff
@@ -67,6 +71,21 @@ class backup_test extends advanced_testcase {
     /** @var string for the testcase, must NOT be a real server! */
     private $apiurl = 'http://server.opencast.testcase';
 
+    /** @var int the episode lti module id */
+    private $episodeltimoduleid = 0;
+     /** @var int the series lti module id */
+    private $seriesltimoduleid = 0;
+    /** @var string old series id */
+    private $oldseriesid = '1234-5678-abcd-efgh';
+    /** @var string old episode id */
+    private $oldepisodeid = 'c0c8c98d-ad90-445c-b1be-be4944779a24';
+    /** @var string new series id */
+    private $newseriesid = '1234-1234-1234-1234';
+    /** @var string new episode id */
+    private $newepisodeid = '66c59fbc-77ca-401a-bb0f-3362bcd27b62';
```

Comment:
Thanks for the suggestion, but does that make a difference?
I would prefer to keep that align with apiurl

Reply:
Not really, just that I thought it might generally be good practice to use `const` when the content of variables doesn't/shouldn't change (apiurl could of course be a const to), so that the variable cannot be used incorrectly, and also the IDE would complain instantly. But it also would be okay for me if it stays like this.